3rdParty headers are missing

When building Apache Metron Kafka plugin against Zeek 3.0.0, the 3rdparty headers to support JSON formatting are missing.

Compiler output:
```
Building CXX object CMakeFiles/APACHE-KAFKA.linux-x86_64.dir/src/KafkaWriter.cc.o
/usr/bin/c++   -DAPACHE_KAFKA_linux_x86_64_EXPORTS -DZEEK_PLUGIN_INTERNAL_BUILD=false -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches   -m64 -mtune=generic  -std=c++11 -Wall -Wno-unused -Wall -Wno-unused -O2 -g -DNDEBUG -fPIC -I/builddir/build/BUILD/metron-bro-plugin-kafka-43c9166787649e4ac2ab295a1baba94d54903651/src -I/builddir/build/BUILD/metron-bro-plugin-kafka-43c9166787649e4ac2ab295a1baba94d54903651 -I/builddir/build/BUILD/metron-bro-plugin-kafka-43c9166787649e4ac2ab295a1baba94d54903651/build/src -I/builddir/build/BUILD/metron-bro-plugin-kafka-43c9166787649e4ac2ab295a1baba94d54903651/build -I/usr/include/zeek    -o CMakeFiles/APACHE-KAFKA.linux-x86_64.dir/src/KafkaWriter.cc.o -c /builddir/build/BUILD/metron-bro-plugin-kafka-43c9166787649e4ac2ab295a1baba94d54903651/src/KafkaWriter.cc
In file included from /builddir/build/BUILD/metron-bro-plugin-kafka-43c9166787649e4ac2ab295a1baba94d54903651/src/KafkaWriter.h:25:0,
                 from /builddir/build/BUILD/metron-bro-plugin-kafka-43c9166787649e4ac2ab295a1baba94d54903651/src/Plugin.h:21,
                 from /builddir/build/BUILD/metron-bro-plugin-kafka-43c9166787649e4ac2ab295a1baba94d54903651/src/Plugin.cc:18:
/usr/include/zeek/threading/formatters/JSON.h:7:29: fatal error: 3rdparty/json.hpp: No such file or directory
 #include "3rdparty/json.hpp"
                             ^
compilation terminated.
In file included from /builddir/build/BUILD/metron-bro-plugin-kafka-43c9166787649e4ac2ab295a1baba94d54903651/src/KafkaWriter.h:25:0,
                 from /builddir/build/BUILD/metron-bro-plugin-kafka-43c9166787649e4ac2ab295a1baba94d54903651/src/KafkaWriter.cc:18:
/usr/include/zeek/threading/formatters/JSON.h:7:29: fatal error: 3rdparty/json.hpp: No such file or directory
 #include "3rdparty/json.hpp"
                             ^
compilation terminated.
make[2]: *** [CMakeFiles/APACHE-KAFKA.linux-x86_64.dir/src/Plugin.cc.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [CMakeFiles/APACHE-KAFKA.linux-x86_64.dir/src/KafkaWriter.cc.o] Error 1
make[2]: Leaving directory `/builddir/build/BUILD/metron-bro-plugin-kafka-43c9166787649e4ac2ab295a1baba94d54903651/build'
make[1]: *** [CMakeFiles/APACHE-KAFKA.linux-x86_64.dir/all] Error 2
make[1]: Leaving directory `/builddir/build/BUILD/metron-bro-plugin-kafka-43c9166787649e4ac2ab295a1baba94d54903651/build'
make: *** [all] Error 2
```

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -440,7 +440,10 @@ install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/
         PATTERN "CMakeFiles" EXCLUDE
 )
 
-install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/sqlite3.h
+install(FILES 
+        ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/fifo_map.hpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/json.hpp
+        ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/sqlite3.h
         DESTINATION include/zeek/3rdparty
 )
 

